src/components/Modal.tsx

```diff
@@ -5,7 +5,7 @@ import Button from './Button'
 import { BUTTONS_THEME } from '../constants/data'
 import { COLORS } from '../constants/colors'
 import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
-import AsyncStorage from '../utils/AsyncStorage'
+import AsyncStorage from '../services/AsyncStorage'
```

Comment:
In integreat we'll do the other way, currently its services but we are planning on changing it to utils as almost none are real services.

Reply:
hmh, yeah thought about this also. So maybe i keep it as it was. Not sure though.